tests/test_pdf_objects.py

```diff
@@ -59,28 +59,21 @@ class TestPdfObjects(unittest.TestCase):
     def setUp(self):
         self.test_file = open(TEST_PDF, 'rb')
         self.pdf_file_object = parse_pdf_document(self.test_file)
-        self.pdf_file_object_2 = parse_pdf_document_pdftxt(self.test_file)
 
     def tearDown(self):
         self.test_file.close()
 
-    def test_pdftotext_equal(self):
-        self.assertEqual(
-            len(self.pdf_file_object.pages),
-            len(self.pdf_file_object_2.pages)
-        )
-
     def test_mean(self):
         font_mean = self.pdf_file_object.get_mean_font_size()
-        self.assertEqual(font_mean, 13)
+        self.assertEqual(font_mean, 24)
```

Comment:
It’s not worth changing now, but if you use py.test you’d be able to write much nicer assertions.

Also, why is 24 suddenly the right answer instead of 13? Has the test PDF changed?

https://alexwlchan.net/2016/10/why-i-use-pytest/